cg/apps/tb/api.py

```diff
@@ -42,15 +43,25 @@ def auth_header(self) -> dict:
         jwt_token = jwt.encode(signer=signer, payload=payload).decode("ascii")
         return {"Authorization": f"Bearer {jwt_token}"}
 
-    def query_trailblazer(self, command: str, request_body: dict) -> Any:
+    def query_trailblazer(self, command: str, request_body: dict, method: str = "POST") -> Any:
         url = self.host + "/" + command
         LOG.debug(f"REQUEST HEADER {self.auth_header}")
-        LOG.debug(f"POST: URL={url}; JSON={request_body}")
-        response = requests.post(
-            url=url,
-            headers=self.auth_header,
-            json=request_body,
-        )
+
+        if method == "PUT":
```

Comment:
Instead of this if tree - you could make this into a dispatch table. See cg/io/controller.py for inspiration